gems/sorbet-runtime/lib/types/props/unknown_value.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+# typed: strict
+
+module T::Props
+  class UnknownValue
+    extend T::Sig
+    extend T::Helpers
+
+    sig {returns(T.untyped)}
+    attr_reader :serialized_value
+
+    sig {returns(StandardError)}
+    attr_reader :error
+
+    sig {params(serialized_value: T.untyped, error: StandardError).void}
+    def initialize(serialized_value, error)
+      @serialized_value = T.let(serialized_value, T.untyped)
+      @error = T.let(error, StandardError)
+    end
```

Comment:
Very minor: WDYT about making `new` a private method, and instantiating this via `make_only_from_with_sorbet_runtime` (or other similarly-named "don't use this" method)?